portfolio/src/main/webapp/script.js

```diff
@@ -94,9 +94,21 @@ window.addEventListener('scroll', function() {
 })
 
 /**
+ * Gets message from DataServlet by fetching a response.
  */
 function getMessage() {
-  fetch('/data').then(response => response.text()).then((message) => {
-    document.getElementById('message-container').innerText = message;
+  fetch('/data').then(response => response.json()).then((message) => {
+    const element = document.getElementById('message-container')
+    element.innerHTML = '';
+    element.appendChild(createListElement(message.string1));
```

Comment:
Optional - It might be worth only calling appendChild if the 'message.stringx' exists, to be robust to errors or changes in the servlet.

Reply:
The messages are hard-coded for the moment, it's just to see that it works. The following tutorials will probably do something to prevent those errors when the messages are actually requested from a server, so I think it would be better to wait after step 4 and 5 to see if this still needs to be implemented.